### PR TITLE
add life time example

### PR DESCRIPTION
**Pull request by boundless-forest**

### CODE REVIEW

**Comment by boundless-forest** [comment]
https://gist.github.com/ouwenkg/a47fbbc8b0e33c5200cc488f1dc4fa15

File: LifeTimeExample.sol [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+        owner = msg.sender;
+        freeze = false;
+    }
+
+    modifier CheckFreeze {
+        require(freeze == false, "This function is freezed.");
+        _;
+    }
+
+    function setFreeze() public {
```
Comment (kaikai1024) [inline LifeTimeExample.sol]: Just `Freeze` and `Unfreeze`

File: LifeTimeExample.sol [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+    function setUnfreeze() public {
+        freeze = false;
+    }
+
+    function set(uint x) CheckFreeze public {
+        storedData = x;
+    }
+
+    function get() view public returns (uint) {
```
Comment (kaikai1024) [inline LifeTimeExample.sol]: Follow the style guide:
This fuction should at the next of `kill`

File: LifeTimeExample.sol (hunk 1 above)
Comment (boundless-forest) [reply LifeTimeExample.sol]: OK

any more advices?

**Comment by kaikai1024** [comment]
And update the readme

File: LifeTimeExample.sol (hunk 1 above)
Comment (kaikai1024) [reply LifeTimeExample.sol]: That's all.

Comment (boundless-forest) [reply LifeTimeExample.sol]: `fn freeze()` have the same name with `bool public freeze`, so i design setXxx

Comment (kaikai1024) [reply LifeTimeExample.sol]: `bool freeze` should be `bool frozen`

File: LifeTimeExample.sol (hunk 2 above)
Comment (boundless-forest) [reply LifeTimeExample.sol]: Done!

**Review by kaikai1024 — APPROVED** [review]

**Pull request merged**